.golangci.yaml

```diff
@@ -5,27 +5,26 @@ linters:
     - revive
     - gosec
     - misspell
-    - dupl
     - exportloopref
     - dupl
+    - gosimple
+    - staticcheck
 
 linters-settings:
+  gofmt:
+    simplify: true
   govet:
     check-shadowing: true
   gci:
     local-prefixes: github.com/rancher/distros-test-framework
-  gosec:
-    issues:
-      - G306
-      - G106
```

Comment:
Are these removed because the framework is hitting edge cases and failing to adhere to these rules?

Reply:
it was removed not on purpose , going back